WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -426,83 +388,20 @@ class MySiteViewModelTest : BaseUnitTest() {
     }
 
     /* ON RESUME */
-
-    @Test
-    fun `given not first resume and phase 2 disabled, when on resume, then update site settings if necessary`() {
-        whenever(mySiteDashboardPhase2FeatureConfig.isEnabled()).thenReturn(false)
-
-        viewModel.onResume(false)
-
-        verify(selectedSiteRepository).updateSiteSettingsIfNecessary()
-    }
-
-    @Test
-    fun `given not first resume and phase 2 disabled, when on resume, then refresh quick start`() {
-        whenever(mySiteDashboardPhase2FeatureConfig.isEnabled()).thenReturn(false)
-
-        viewModel.onResume(false)
-
-        verify(quickStartCardSource).refresh()
-    }
-
-    @Test
-    fun `given not first resume and phase 2 disabled, when on resume, then refresh current avatar`() {
-        whenever(mySiteDashboardPhase2FeatureConfig.isEnabled()).thenReturn(false)
-
-        viewModel.onResume(false)
-
-        verify(currentAvatarSource).refresh()
-    }
-
-    @Test
-    fun `given first resume and phase 2 disabled, when on resume, then update site settings if necessary`() {
-        viewModel.onResume(true)
-
-        verify(selectedSiteRepository).updateSiteSettingsIfNecessary()
-    }
-
-    @Test
-    fun `given first resume and phase 2 disabled, when on resume, then refresh quick start`() {
-        viewModel.onResume(true)
-
-        verify(quickStartCardSource).refresh()
-    }
-
-    @Test
-    fun `given first resume and phase 2 disabled, when on resume, then refresh current avatar`() {
-        viewModel.onResume(true)
-
-        verify(currentAvatarSource).refresh()
-    }
-
     @Test
-    fun `given first resume and phase 2 enabled, when on resume, then update site settings if necessary`() {
-        viewModel.onResume(true)
+    fun `given not first resume, when on resume is triggered, then mySiteSourceManager onResume is invoked`() {
+        val firstResume = true
```

Comment:
Minor (🔍): Consider living a space between the `given` and `when` actions.

PS: The same applies to the below test.